apps/masterbots.ai/lib/hooks/use-thread-visibility.tsx

```
@@ -0,0 +1,190 @@
+}
+
+export function ThreadVisibilityProvider({ children }: ThreadVisibilityProviderProps) {
+  const [isPublic, setIsPublic] = useState(false);
+  const [threads, setThreads] = useState<Thread[]>([]);
+  const [isAdminMode, setIsAdminMode] = React.useState<boolean>(false);
+
+  const session = useSession();
+  const jwt = session?.data?.user?.hasuraJwt;
+
+
+  useEffect(() => {
```

Biome

[error] This hook specifies more dependencies than necessary: jwt

This dependency can be removed from the list.


(lint/correctness/useExhaustiveDependencies)